pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -356,6 +356,8 @@ class Build {
             def type = "jdk"
             if (file.contains("-jre")) {
                 type = "jre"
+            } else if (file.contains("-testimage")) {
```

Comment:
This is the change needed so that `foo.tar.gz.json` metadata gets `testimage` for `binary_type`. It would be `jdk` otherwise.